docs/rules.md

```diff
@@ -363,6 +364,18 @@ These are considered invalid:
     end
 
 
+### NoInterpolationOnlyStrings
+
+A rule that disallows strings which are entirely the result of an
+interpolation.
+
+Good:
+    output = "Dogma.Rule.NoInterpolationOnlyStrings"
```

Comment:
Looks like part of the moduledoc is being interpreted... Expected? Should I just pick a more innocuous thing to inspect in my example?